src/Features/Core/Portable/GoToDefinition/AbstractGoToDefinitionSymbolService.cs

```diff
@@ -68,18 +76,5 @@ public async Task<(int? targetPosition, TextSpan tokenSpan)> GetTargetIfControlF
 
             return (GetTargetPositionIfControlFlow(semanticModel, token), token.Span);
         }
-
-        private static ISymbol? GetSymbol(TokenSemanticInfo semanticInfo, bool includeType)
-        {
-            // Prefer references to declarations. It's more likely that the user is attempting to 
-            // go to a definition at some other location, rather than the definition they're on. 
-            // This can happen when a token is at a location that is both a reference and a definition.
-            // For example, on an anonymous type member declaration.
-
-            return semanticInfo.AliasSymbol
-                ?? semanticInfo.ReferencedSymbols.FirstOrDefault()
-                ?? semanticInfo.DeclaredSymbol
-                ?? (includeType ? semanticInfo.Type : null);
```

Comment:
this logic seemed to serve no purpose.  And it meant ctrl-click and go-to-def were not in sync.  i removed it.